manual_user_install.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+# tested with a clean 64 bit installation of Raspbian-Lite
+
+function install_core (){
+    echo "Installing OVOS core"
+    echo
+    pip3 install -U pip
+
+    # install ovos-core - ovos-backend-client has conflicting dependencies
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-backend-client
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-core
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-audio
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin
+    pip3 install tornado
```

Comment:
not needed, ovos-messagebus will install tornado itself

Reply:
Ha!! I thought you mentioned the `pip install-U pip' command, not `ovos-backend-client` that should  and needs to install, correct?

Reply:
tornado removed from my copy. Will test.

Reply:
it works